packages/website/next.config.js

```diff
@@ -5,4 +5,18 @@ const withMDX = require('@next/mdx')({
 
 module.exports = withMDX({
   pageExtensions: ['ts', 'tsx', 'mdx'],
+  async redirects() {
+    return [
+      {
+        source: '/login',
+        destination: '/api/auth/signin',
+        permanent: true,
+      },
+      {
+        source: '/signin',
```

Comment:
Do we need both `/login` and `/signin`?

Reply:
I though that it would be nice to have both since we don't have any link/button to the login page. Do you think it's overkill?

Reply:
I think that keeping only one would be fine

Reply:
done, thanks for the feedback!